### PR TITLE
pdf proxy try

### PR DESCRIPTION
**Pull request by elbotho**

### CODE REVIEW

File: src/pdf-proxy.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  try {
+    url.hostname = 'pdf.serlo.org'
+    url.pathname = `/api/${pdfMatch[1]}`
+
+    const pdfRes = await fetch(url.href, {
+      cf: { cacheTtl: 24 * 60 * 60 },
+    } as unknown as RequestInit)
+
+    if (pdfRes.ok) return pdfRes
+  } catch (e) {
```
Comment (kulla) [inline src/pdf-proxy.ts:46]: There is no possibility for an exception to be thrown -> delete it (maybe we want to add a global try-catch to handle uncatched exceptions)

Comment (kulla) [reply src/pdf-proxy.ts:46]: Do you want to report for non 200 responses?!

**Review by kulla — CHANGES_REQUESTED** [review]

File: src/pdf-proxy.ts (hunk 1 above)
Comment (elbotho) [reply src/pdf-proxy.ts:46]: tbh. it's just copy&paste from the `quickbar-proxy`.
Probably same case there. I guess 500 or 404 are unlikely but worth reporting.

Comment (kulla) [reply src/pdf-proxy.ts:46]: Yeah, you can also report any other response and narrow it down later when you false positives in sentry...

**Review by kulla — APPROVED** [review]

**Pull request merged**